package/android/cpp/rnskia-android/SkiaOpenGLRenderer.h

```diff
@@ -26,96 +28,44 @@
 #pragma clang diagnostic pop
 
 namespace RNSkia {
-sk_sp<SkSurface> MakeOffscreenGLSurface(int width, int height);
-
-using OpenGLDrawingContext = struct {
-  EGLContext glContext;
-  EGLDisplay glDisplay;
-  EGLConfig glConfig;
-  sk_sp<GrDirectContext> skContext;
-};
 
-static std::unordered_map<std::thread::id,
-                          std::shared_ptr<OpenGLDrawingContext>>
-    threadContexts;
+/**
+ * The first context created will be considered the parent / shared context and
+ * will be used as the parent / shareable context when creating subsequent
+ * contexts.
+ */
+static EGLContext sharedEglContext = EGL_NO_CONTEXT;
```

Comment:
this variable doesn't appear to have any effect nor being a shared context (EGL_NO_CONTEXT means that no context will be shared). Is `sharedEglContext ` updated somewhere?